src/main/java/com/commercetools/sync/sdk2/commons/utils/CollectionUtils.java

```diff
@@ -119,46 +95,11 @@ public static <T> Collection<T> emptyIfNull(@Nullable final Collection<T> collec
    * @param <T> list entities type
    * @return original {@code list} instance, if non-null; otherwise immutable empty list instance.
    * @see #emptyIfNull(Collection)
-   * @see #emptyIfNull(Set)
-   * @see #emptyIfNull(Map)
    */
   @Nonnull
   public static <T> List<T> emptyIfNull(@Nullable final List<T> list) {
     return list == null ? emptyList() : list;
   }
 
-  /**
-   * Safe wrapper around nullable set instances: returns {@code set} argument itself, if the {@code
-   * set} is non-null, otherwise returns (immutable) empty set.
-   *
-   * @param set {@link Set} instance to process
-   * @param <T> set entities type
-   * @return original {@code set} instance, if non-null; otherwise immutable empty set instance.
-   * @see #emptyIfNull(Collection)
-   * @see #emptyIfNull(List)
-   * @see #emptyIfNull(Map)
-   */
-  @Nonnull
-  public static <T> Set<T> emptyIfNull(@Nullable final Set<T> set) {
```

Comment:
If we remove public methods, we should mention it in the migration notes. MAYBE someone is using it 😅

Reply:
[7a5d882](https://github.com/commercetools/commercetools-sync-java/pull/1060/commits/7a5d882523740e5abca84f504568a6e0131d90d4)